Documentation/Conditions/Index.rst

```diff
@@ -747,6 +747,12 @@ request.getPageArguments()
    
    Allows migration from old condition syntax using `[globalVar = GP:singlepartner > 0]`
    to `[request.getPageArguments().get('singlepartner') > 0]`.
+   
+   
+Functions in all contexts
+-------------------------
+
+The following functions are available in **all** contexts: frontend, backend and CLI
```

Comment:
```suggestion
The following functions are available in **any** context:
```

Reply:
There is also the install tool context, so we'd better not make a list here.

Reply:
didn't know that - makes sense!